pymongo/asynchronous/mongo_client.py

```diff
@@ -1517,6 +1521,9 @@ async def close(self) -> None:
             # TODO: PYTHON-1921 Encrypted MongoClients cannot be re-opened.
             await self._encrypter.close()
 
+    async def aclose(self) -> None:
+        await self.close()
```

Comment:
Public API should have a docstring.

Reply:
Why do we need aclose() when we already have close()?

Reply:
`contextlib` expects that asynchronous context managers use `aclose`. We could simplify by just setting `aclose = close` instead.

Reply:
In that case should we remove "close()"?

Reply:
I think that's better, yeah.